Mathlib/CategoryTheory/Monoidal/Functorial.lean

```diff
@@ -47,36 +47,69 @@ open MonoidalCategory
 variable {C : Type u₁} [Category.{v₁} C] [MonoidalCategory.{v₁} C] {D : Type u₂} [Category.{v₂} D]
   [MonoidalCategory.{v₂} D]
 
--- Perhaps in the future we'll redefine `LaxMonoidalFunctor` in terms of this,
--- but that isn't the immediate plan.
-/-- An unbundled description of lax monoidal functors. -/
-class LaxMonoidal (F : C → D) [Functorial.{v₁, v₂} F] where
+/-- An unbundled description of lax monoidal functors without axioms. See `LaxMonoidal` for
+the full description. -/
+class LaxMonoidalStruct (F : C → D) [Functorial.{v₁, v₂} F] where
```

Comment:
Is this refactor something that could be split out separately?

Reply:
@eric-wieser I noticed that `LaxMonoidalStruct` is essentially only used in `LaxMonoidal.ofTensorHom`, which is another constructor for `LaxMonoidal` using `tenroHom` instead of the whiskerings, to avoid tautological transport for `ε` and `μ`. I now think we don't need this type class at this point because it has a tiny benefit, so I have reverted the addition of this type class for now.